go-controller/go.mod

```diff
@@ -43,6 +43,7 @@ require (
 	k8s.io/api v0.25.2
 	k8s.io/apimachinery v0.25.2
 	k8s.io/client-go v0.25.0
+	k8s.io/klog v1.0.0
```

Comment:
Is there a strict need to use klog v1?

Reply:
I don't think so, probably a mistake on my side
removed the (probably wrong) imports and removed this commit